.changeset/orange-ads-join.md

```diff
@@ -2,10 +2,20 @@
 "@fluidframework/container-runtime": minor
 "@fluidframework/runtime-definitions": minor
 ---
+---
+section: deprecation
+---
+
+gcThrowOnTombstoneUsage and gcTombstoneEnforcementAllowed are deprecated
```

Comment:
@agarwal-navin These are the changeset changes I made. To preview the release notes, I recommend viewing them directly on my PR branch here: https://github.com/tylerbutler/FluidFramework/blob/relnotes-2.2/RELEASE_NOTES/2.2.0.md